Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -1112,14 +1121,26 @@ def list_mails_command(client: MsGraphClient, args):
     odata = args.get('odata')
 
     raw_response = client.list_mails(user_id, folder_id=folder_id, search=search, odata=odata)
+    last_page_response = raw_response[len(raw_response)-1]
+    metadata = ''
+    next_page = last_page_response.get('@odata.nextLink')
+    if next_page:
+        metadata = f'\nPay attention there are more results than shown. For more data please ' \
+                               f'increase "pages_to_pull" argument'
```

Comment:
by how much? do you get the total amounts of mails?
we can discuss it, but in general, we prefer that each time the cmd run x(20) mails will be pulled.
so e.g:
`!ms-graph-mail-list-emails pages_to_pull=1 page_size =2` - will return the first 20.
`!ms-graph-mail-list-emails pages_to_pull=2 page_size =2` - will return the 20 to 40 emails.

Reply:
I dont get the total amount of results, and your example is exactly the logic, pages_to_pull argument implemented before my change and this is exactly what he do- it loop over the number of pages and concatenate